### PR TITLE
chore: Update `@nextcloud/vue`

### PR DESCRIPTION
**Pull request by susnux**
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: dist/core-common.js.LICENSE.txt [hunk 1]
```diff
@@ -585,6 +585,28 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
 /**!
  * @fileOverview Kickass library to create and place poppers near their reference elements.
```
Comment (ChristophWurst) [inline dist/core-common.js.LICENSE.txt:611]: :eyes:

**Review by AndyScherzinger — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
Tested and works

image

^ found some unexpected white space left of the favorite icon in the sidebar but the same issue exists on master.

**Review by JuliaKirschenheuter — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by blizzz** [comment]
`theming/admin-settings.cy.ts` is still expected to fail, right?

**Comment by susnux** [comment]
> `theming/admin-settings.cy.ts` is still expected to fail, right? Cypress.

Yes the fixing PR is still not merged

**Pull request merged**